test/integration/test_noise_learner.py

```diff
@@ -0,0 +1,43 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Integration tests for NoiseLearner."""
+
+from qiskit.circuit import QuantumCircuit
+from qiskit_ibm_runtime import NoiseLearner
+
+from ..decorators import run_integration_test
+from ..ibm_test_case import IBMIntegrationTestCase
+
+
+class TestIntegrationNoiseLearner(IBMIntegrationTestCase):
+    """Integration tests for NoiseLearner."""
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.backend = "ibmq_qasm_simulator"
```

Comment:
this should use `test_eagle_us-east` instead since we don't have cloud simulators (on IQP) anymore.